Infrastructure-Scripts/Set-AppRoleAssignments/tools/Helpers.psm1

```diff
@@ -65,9 +66,20 @@ function Set-AzureCLIAccess {
 
     az rest @MicrosoftGraphRequestParameters
 
-    #https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#use-quotation-marks-in-arguments
+    #Set signInAudience to AzureADMyOrg to be able to use v1.0 tokens ie set requestedAccessTokenVersion to 1
+    #https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens#payload-claims
+    #https://docs.microsoft.com/en-us/graph/api/application-update?view=graph-rest-1.0&tabs=http
+    $MicrosoftGraphRequestParameters =
+    "--method", "patch",
+    "--uri", "https://graph.microsoft.com/v1.0/applications/$AppRegistrationObjectId",
+    "--headers", "Content-Type=application/json",
+    "--body", "{'signInAudience' : 'AzureADMyOrg'}"
+
+    az rest @MicrosoftGraphRequestParameters
```

Comment:
Can't this be set when we create the app instead? Saves an extra call

Reply:
Also functionally, our services probably need v1 tokens as well, not sure this is specific to the AzureCLI access requirement

Reply:
Have made those changes 🔥